src/helix-ui/elements/HXElement.js

```diff
@@ -85,6 +85,11 @@ export class HXElement extends HTMLElement {
     }//$preventScroll()
 
     $emit (evtName, details) {
+        if (window.ShadyCSS && evtName === 'change') {
+            // Let native 'change' events bubble naturally.
+            return;
+        }
```

Comment:
(2 of 2) Fix compatibility issue with regards to native `change` events in polyfilled browsers.